scripts/kiwix-compile-i18n

```diff
@@ -61,7 +61,7 @@ lang_table_entry_cxx_template = '''
 
 cxxfile_template = '''// This file is automatically generated. Do not modify it.
 
-#include "server/i18n.h"
```

Comment:
This first commit introduce the file `test/data/multilang.zim`. It should not.

Reply:
Thanks for catching this. It has been fixed.